x-pack/plugins/security_solution/server/lib/telemetry/tasks/prebuilt_rule_alerts.ts

```diff
@@ -115,7 +107,6 @@ export function createTelemetryPrebuiltRuleAlertsTaskConfig(maxTelemetryBatch: n
         }
 
         taskMetricsService.end(trace);
-        await receiver.closePointInTime(pitId);
```

Comment:
No longer need the PIT stuff because we're paginating right?

Reply:
yes, the PIT logic is managed by `fetchPrebuiltRuleAlertsBatch`